entrypoint.sh

```diff
@@ -12,6 +12,10 @@ optionals+=""
 [[ -S ${CRI_RUNTIME_ENDPOINT} ]] && optionals+="-cri-runtime-endpoint ${CRI_RUNTIME_ENDPOINT} "
 [[ "${PAUSE_IMAGE}" ]] && optionals+="-pause-image ${PAUSE_IMAGE} "
 [[ "${VXLAN_PORT}" ]] && optionals+="-vxlan-port ${VXLAN_PORT} "
+[[ "${CACERT_FILE}" ]] && optionals+="-ca-cert-file ${CACERT_FILE} "
+[[ "${CERT_FILE}" ]] && [[ "${CERT_KEY}" ]] && optionals+="-cert-file ${CERT_FILE} -cert-key ${CERT_KEY} "
+[[ "${CERT_KEY}" ]] && optionals+="-cert-key ${CERT_KEY} "
```

Comment:
Is it needed here? now if you set CERT_FILE and CERT_KEY the cmdline will look as follow:
`-cert-file ${CERT_FILE} -cert-key ${CERT_KEY} -cert-key ${CERT_KEY} `

Reply:
Good catch. Updated it